src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/LocationSelectPostRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace SFA.DAS.LevyTransferMatching.Web.Models.Pledges
+{
+    public class LocationSelectPostRequest : LocationSelectRequest
+    {
+        public LocationSelectionGroup[] LocationSelectionGroups { get; set; }
```

Comment:
I think these properties could be named more clearly - perhaps to indicate that each "group" relates to an ambiguous location input, and each of the "items" are possible valid alternatives? I don't really get that from group/item.

Reply:
Renamed sub-classes to `SelectValidLocationGroup` and `ValidLocationItem` respectively. Not an easy one to name, tbh.